docs/mmteb/readme.md

```diff
@@ -0,0 +1,64 @@
+# Welcome to MMTEB! 👋
+
+MMTEB is a community-led extension of [MTEB](https://arxiv.org/abs/2210.07316) to cover embedding tasks for a massive number of languages.
+
+## Background 
+
+MTEB or the Massive Text Embedding Benchmark is intended to evaluate the quality of document embeddings. When it was initially introduced, the benchmark consisted of 8 embedding tasks and 58 different datasets. Since then, MTEB has been subject to multiple community contributions as well as benchmark extensions over specific languages such as [SEB](https://openreview.net/pdf/f5f1953a9c798ec61bb050e62bc7a94037fd4fab.pdf), [C-MTEB](https://github.com/FlagOpen/FlagEmbedding/tree/master/C_MTEB) and [MTEB-French](https://github.com/Lyon-NLP/mteb-french). However, we want even wider coverage and thus announce the community-led extension of MTEB, where we seek to expand coverage of MTEB to as many languages as possible.
+
+## Contributing to MMTEB
+
+Everyone can join and contribute to this initiative from 1 April 2024 to 31 April 2024, win some SWAG, and become a co-author of our upcoming paper. We aim to publish the results of our findings at a top conference such as EMNLP, NeurIPS, etc. We have identified four tasks for contributions:
+
+### 🗃️ Task 1: Contribute a new dataset
+
+For this segment, you open a PR in the MTEB repository where you create a new instance (subclass) of a task using a new language dataset (we also accept code!). Read more about how to add a task [here](adding_a_new_dataset.md) and check out [one of the previous additions](https://github.com/embeddings-benchmark/mteb/pull/247) for an example.
+
+### 🖥️ Task 2: Contribute a new task
+
+MTEB currently consists of 8 embedding tasks including tasks such as STS, retrieval, reranking, and more. If you feel like there is a category of tasks that is not yet covered, we would welcome contributions of these as well.
+
+### 🔍 Task 3: Contribute new scores
+
+Once we have the datasets, we want to evaluate models on them. We welcome evaluation scores for models, which will be added to the leaderboard.
+
+### 🔓 Task 4: Review PRs
+
+We welcome reviews of PRs adding new datasets. If you wish to review PRs of a specific language feel free to contact members of the MTEB team.
+
+## Authorship
+
+We follow a similar approach as in the [SeaCrowd Project](https://github.com/SEACrowd#contributing-to-seacrowd) and use a point-based system to determine co-authorships. 
+
+To be considered a co-author, at least 10 contribution points are required. Position of contributors in the author list is determined by the score they acquire, higher scores will appear first.
```

Comment:
We might want to contribute points for ideation, repo maintenance and prior work before launching. This would likely have to be a substantial amount of points if we want to ensure that noone is adding ~100 datasets and ends up first author, without having contributed anything else (if that is what we want).

Reply:
I would like this as well. Let us start with the easy stuff (in another PR):

@imenelydiaker: will you submit the points for the french team
@MartinBernstorff: will you give a suggestion of points based on your work
@Muennighoff: Is there people we should get added to the points before we go live?

Just to make clear, we see this as a suggestion to open the discussion, but I would like to make it such that it is transparent for contributors.

let me know if you guys disagree

Reply:
I discussed it with the French team, they said that since we already have a paper it won't count as a contribution to mmteb. They say we should only be added in the references or acknowledgments. 

I wonder how about the other languages that are already in MTEB? I don't know if you guys have any thoughts about it?